### PR TITLE
Travis CI: Explain why we use "cpp" instead of "python"

### PR DESCRIPTION
**Pull request by sschuberth**
Because that deserves some explanation.

### CODE REVIEW

**Comment by pombredanne** [comment]
@sschuberth I reckon it looks odd. But this is NOT a bug. We have to use the `cpp` config because we are using the multi-os (linux + mac) travis support (which is enabled only by special request)  per https://github.com/nexB/license-expression/blob/025fee38d84aa19f32050c0802eb27e0f0852bc7/.travis.yml#L3 and this has to be used then.  So if you do not do that, whatever the internal reasons may be at Travis the mac build will not work at all .... which is what you see just here in the build results

**Comment by sschuberth** [comment]
@pombredanne I was assuming something like that after seeing the OSX builds fail. I'll change this PR to add a comment to `.travis.yml` instead.

**Comment by sschuberth** [comment]
@pombredanne Does the wording of the comment nail it?

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# This deliverately is not "python" as a work-around to support
```
Comment (pombredanne) [inline .travis.yml]: I guess you meant `deliberately`?

Comment (sschuberth) [inline .travis.yml]: Indeed. Thanks for fixing it already.

**Comment by pombredanne** [comment]
Thanks! I tried the "commit in your branch feature" and it works fine.

**Pull request merged**